packages/server/src/koaJsonRpc/lib/methodConfiguration.ts

```diff
@@ -24,12 +24,20 @@ dotenv.config({ path: path.resolve(__dirname, '../../../../../.env') });
 
 import CONSTANTS from '../../../../relay/dist/lib/constants';
 
-const tier1rateLimit = process.env.TIER_1_RATE_LIMIT || CONSTANTS.DEFAULT_RATE_LIMIT.TIER_1;
-const tier2rateLimit = process.env.TIER_2_RATE_LIMIT || CONSTANTS.DEFAULT_RATE_LIMIT.TIER_2;
-const tier3rateLimit = process.env.TIER_3_RATE_LIMIT || CONSTANTS.DEFAULT_RATE_LIMIT.TIER_3;
+const tier1rateLimit = parseInt(process.env.TIER_1_RATE_LIMIT ?? CONSTANTS.DEFAULT_RATE_LIMIT.TIER_1.toString());
```

Comment:
Q: this changes the data type from string to int right?
Was something broken here or what's the benefit of the change?
Maybe it avoids casting later?

Reply:
Yes, it avoids implicit casting later and potential issues which might arise from the ambiguity in the type.

Reply:
Thanks